proto/wechaty/puppet/base.proto

```diff
@@ -36,8 +36,8 @@ message DingResponse {
 }
 
 message DirtyPayloadRequest {
-  PayloadType payloadType = 1;
-  string payloadId = 2;
+  PayloadType payload_type = 1;
```

Comment:
I just reviewed other names, let's just use `type` and `id` in this request to follow the same naming convention as others.

Reply:
Changed.

Reply:
Thank you very much!